src/common/checksum.c

```diff
@@ -78,6 +78,8 @@ daos_csummer_init(struct daos_csummer **obj, struct hash_ft *ft,
 
 	if (rc == 0)
 		*obj = result;
+	else
+		D_FREE_PTR(result);
```

Comment:
FYI, D_FREE_PRT() will be removed at some point, please use D_FREE() instead.

Reply:
Sorry, missed that. I will refresh the patch.